packages/nocodb/src/utils/api-v3-data-transformation.builder.ts

```
@@ -0,0 +1,592 @@
+  transformFn?: (data: any) => any;
+  nestedExtract?: Record<string, string[]>;
+  excludeNullProps?: boolean;
+  booleanProps?: string[];
+  meta?: {
+    snakeCase?: boolean;
+    camelCase?: boolean;
+    mappings?: Record<string, string>;
+    metaProps?: string[];
+    skipTransformFor?: string[];
+    skipfn?: (data: any) => boolean;
+  } & ({ allowed: string[] } | { excluded: string[] } | {});
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)